MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -191,21 +191,26 @@ public class CarPlayMapViewController: UIViewController {
         }
         camera.pitch = 60
         mapView.setCamera(camera, animated: animated)
-
     }
     
     override public func viewSafeAreaInsetsDidChange() {
-        mapView.setContentInset(mapView.safeArea, animated: false)
+        super.viewSafeAreaInsetsDidChange()
+        
+        var edgePadding = view.safeArea
+        edgePadding += NavigationMapView.defaultPadding
         
-        guard isOverviewingRoutes else {
-            super.viewSafeAreaInsetsDidChange()
-            return
+        if let userCourseView = mapView.userCourseView {
+            let midX = userCourseView.bounds.midX
+            let midY = userCourseView.bounds.midY
+            edgePadding += UIEdgeInsets(top: midY, left: midX, bottom: midY, right: midX)
```

Comment:
This if block seems to be ensuring that the course view is always fully visible on the map. But about the waypoints? And if we’re previewing a route where the route line might extend in all directions from the user’s current location, then this alone won’t ensure that the whole route fits on the map. We have logic elsewhere about resetting or fitting the camera. Maybe we should reuse it instead of trying to reimplement some of it via edge insets.